DuckDuckGo/AppUserDefaults.swift

```diff
@@ -21,6 +21,10 @@ import Foundation
 import Core
 
 public class AppUserDefaults: AppSettings {
+    
+    public struct Notifications {
+        public static let doNotSellStatusChange = Foundation.Notification.Name("com.duckduckgo.app.DoNotSellStatusChange")
```

Comment:
Also @samsymons is doing something with notifications to trigger a refresh when the content blocker data changes.  I think you can probably share the same concept.  Probably worth having a chat.

Reply:
Looking at @samsymons PR I'm not sure where this would benefit this case although maybe I'm missing it. Since this setting is independent and does not relate to content blocking I feel having it as its own notification makes sense. I'm open to other opinions though.